lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -39,7 +43,11 @@ sub new {
         partition_table_shortcut        => $args->{partition_table_shortcut},
         clone_partition_chortcut        => $args->{clone_partition_chortcut},
         ok_clone_shortcut               => $args->{ok_clone_shortcut},
-        available_target_disks_shortcut => $args->{avail_tgt_disks_shortcut}
+        available_target_disks_shortcut => $args->{avail_tgt_disks_shortcut},
```

Comment:
the framework is quite verbose on naming, better `available` than `avail` to be consistent.

Reply:
Actually, that's my bad. It was part of https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9211/files.

Reply:
anyway we should change it for consistency, but no big deal. We can leave it as it is.